nbstripout/_nbstripout.py

```diff
@@ -253,6 +261,17 @@ def install(git_config, install_location=INSTALL_LOCATION_LOCAL, attrfile=None):
 
         sys.exit(1)
 
+    with open(attrfile, 'a', newline='') as f:
+        # If the file already exists, ensure it ends with a new line
+        if f.tell():
+            f.write('\n')
+        if not filt_exists:
+            print('*.ipynb filter=nbstripout', file=f)
+        if not zeppelin_filt_exists:
```

Comment:
This rebase seems to have gone wrong: please move the Zeppelin related option to line 254